dspace-server-webapp/src/test/java/org/dspace/app/rest/authorization/CanCreateVersionFeatureIT.java

```diff
@@ -144,13 +144,14 @@ public void submitterItemSuccessTest() throws Exception {
                          .andExpect(status().isOk())
                          .andExpect(jsonPath("$.page.totalElements", greaterThan(0)))
                          .andExpect(jsonPath("$._embedded").exists());
+
+        configurationService.setProperty("versioning.submitterCanCreateNewVersion", false);
```

Comment:
You don't need this line, as the value resets after every test.  Just change the value at the beginning of a test if you need to do so